src/_locales/zh_TW/messages.yml

```diff
@@ -366,6 +366,9 @@ labelMatch:
 labelName:
   description: Label of script name.
   message: 名字：
+labelNoFrames:
+  description: Label of script @noframes properties in custom meta data.
+  message: ''
```

Comment:
@gera2ld is it possible to remove empty translations from these automatic PRs when they're new? I looked at our code but didn't understand why it didn't work: https://github.com/violentmonkey/violentmonkey/blob/083ea2850f5fc97c7cde94b2ea5c19044b37200e/scripts/transifex.js#L143

Reply:
The empty messages are kept intentionally for the translators. They will be replaced with the default messages at build time.
The quoted code block ensures that empty messages are not uploaded to Transifex.